src/Step/ValidateConfiguration.php

```diff
@@ -47,19 +47,20 @@ public function execute(): void
     {
         $this->logger->notice('Validating configuration');
 
-        $messages = $this->collectMessages();
+        $errors = $this->collectErrors();
 
-        ksort($messages);
-        foreach ($messages as $level => $levelMessages) {
-            $error = 'Fix configuration with given suggestions:' . PHP_EOL . implode(PHP_EOL, $levelMessages);
+        ksort($errors);
+        foreach ($errors as $level => $levelErrors) {
+            $message = $this->createErrorMessage($levelErrors);
 
             if ($level >= ValidatorInterface::LEVEL_CRITICAL) {
                 throw new StepException(
```

Comment:
It will be logged but in a different place where `StepException` is catching `\Magento\MagentoCloud\Scenario\Processor::execute`, such behavior was before my changes.